frontend/src/components/LandingPage/Features/Features.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { Media, Button } from 'reactstrap';
+import { Text } from 'react';
+import './Features.css'
+import spacePic from './spacepic.jpg';
+import musicPic from './musicpic.jpg';
+import newsPic from './newspic.jpg';
+import weatherPic from './weatherpic.jpg';
+import factPic from './factpic.jpg';
+
+var picStyle = {
+    maxWidth: "128px",
+    maxHeight: "128px",
+    minHeight: "128px",
+    marginRight: 5,
+    borderTopLeftRadius: 20,
+    borderBottomLeftRadius: 20,
+};
+
+var mediaStyle = {
+    flex: 1,
+    marginLeft: 15,
+    marginTop: 10,
+    backgroundColor: "#7a748c",
+    borderRadius: 20,
+    marginRight: 15,
+    marginBottom: 10,
+};
+
```

Comment:
What do you think about moving the css to be inside of the Features.css ? That way this file will look a bit cleaner.